plaidml2/bridge/keras/backend_test.py

```diff
@@ -525,7 +525,7 @@ def testAddElementsRepeated(self, b, x, y, z):
         return [x + y + z]
 
     @opTest([
-        [m(3, 3), 1.0],
+        [m(3, 3), 65555],
```

Comment:
This probably doesn't belong in this commit

Reply:
sorry. Fixed.